docs/indexer/custom-processors/e2e-tutorial.md

```diff
@@ -73,107 +64,113 @@ For mac, if you're using brew, start it up with:
-        - coin_flip
-            - move
-                - sources
-                    - coin_flip.move
-                    - package_manager.move
-                - Move.toml
-            - config.yaml     <-------- Edit this config.yaml file
-            - models.py
-            - processor.py
-            - README.md
-        - example_event_processor
-        - nft_marketplace_v2
-        - nft_orderbooks
-        __init__.py
-        main.py
-        README.md
+        - processors
+            - aptos_ambassador_token
+            - aptos-tontine
+            - coin_flip
+                - move
+                    - sources
+                        - coin_flip.move
+                        - package_manager.move
+                    - Move.toml
+                - config.yaml     <-------- Create and edit this config.yaml file
+                - models.py
+                - processor.py
+                - README.md
+            - example_event_processor
+            - nft_marketplace_v2
+            - nft_orderbooks
+            __init__.py
+            main.py
+            README.md
     - rust
     - scripts
     - typescript
 ```
 
-Once you have your config.yaml file open, you only need to change one field, if you just want to run the processor as is:
-
+Once you have your config.yaml file open, you only need to update a certain fields
 ```yaml
-grpc_data_stream_api_key: "<YOUR_API_KEY_HERE>"
+server_config:
+    processor_config: 
+        type: "coin_flip"
+    chain_id: 2
+    indexer_grpc_data_service_address: "grpc.testnet.aptoslabs.com:443"
+    auth_token: "<API-KEY>"
+    postgres_connection_string: "postgresql://localhost:5432/<Database Name>"
+    # Optional. Start processor at starting_version
+    starting_version: 636585029
+    # Optional. Stop processor after ending_version.
+    ending_version: 636589723
 ```
 
 ### More customization with config.yaml
 
 However, if you'd like to customize things further, you can change some of the other fields.
 
 If you'd like to start at a specific version, you can specify that in the config.yaml file with:
-
 ```yaml
-starting_version_default: 123456789
+starting_version: <Starting Version>
 ```
 
-This is the transaction version the indexer starts looking for events at. If the indexer has already processed transactions past this version, **it will skip all of them and go to the latest version stored.**
-
-The rows in `next_versions_to_process` are the `indexer_name` as the primary key and the `next_version` to process field, along with the `updated_at`.
+This is the transaction version the indexer starts looking for events at.
 
-If you want to **force** the indexer to backfill data (overwrite/rewrite data) from previous versions even though it's already indexed past it, you can specify this in the config.yaml file with:
+The rows in table `next_versions_to_process` are the `indexer_name` as the primary key and the `next_version` to process field, along with the `updated_at`.
```

Comment:
This sentence, needs some work.

What is it trying to convey? (the old sentence was bad too)

Are we describing how the table is laid out?

Could we say what everything is and what it means in separate sentences for each.  This right now says, there's rows, with things in them, and then there's more things, but without reason.

Perhaps

```suggestion
The table `next_versions_to_process` keeps track of the current state of indexing.  The primary key is the `indexer_name`.  The `next_version` field tells us what's the next transaction to process, and the `updated_at` time tells us the last time the indexer processed a transaction.
```

Reply:
Yep, looks like this sentence was in the doc previously (line 138).

You got it. It was try to describe the info (columns) in the new table `next_versions_to_process`